.github/workflows/test-release.yml

```diff
@@ -0,0 +1,43 @@
+### DO NOT MERGE
```

Comment:
Should this file be removed before merging?

Reply:
Up to you. It could be used to test the release in merge requests. If desired this line should be removed, else the entire file

Reply:
Let's remove the entire file